lib/SparkPost/Transmission.php

```diff
@@ -63,12 +63,22 @@ private function __construct() {}
 	 */
 	private static function getHttpClient() {
 		if(!isset(self::$request)) {
-			self::$request = new Client();
+			$hostConfig = SparkPost::getConfig();
+			$defaultOptions = [
+				'base_uri'=>self::getBaseUrl($hostConfig),
+				'headers' =>[
+					'Authorization' => $hostConfig['key'],
+					'Content-Type' => 'application/json',
+					'User-Agent' => 'SparkPost PHP SDK v0.1.2'
```

Comment:
Yes. I like to keep it consistent with the other SDKs if possible. So `php-sparkpost/0.2.0` if possible.